tests/basic/storage_pool_test.go

```diff
@@ -30,103 +30,49 @@ import (
 )
 
 const (
-	replicationUpdateTimeout = 4 * time.Hour
-	poolResizeTimeout        = time.Minute * 360
-	retryTimeout             = time.Minute * 2
-	addDriveUpTimeOut        = time.Minute * 15
-	maxPoolLength            = 8
+	replicationUpdateTimeout         = 4 * time.Hour
+	retryTimeout                     = time.Minute * 2
+	addDriveUpTimeOut                = time.Minute * 15
+	poolResizeTimeout                = time.Minute * 90
+	poolExpansionStatusCheckInterval = time.Minute * 3
 )
 
-var _ = Describe("{StoragePoolExpandDiskResize}", func() {
-	JustBeforeEach(func() {
-		StartTorpedoTest("StoragePoolExpandDiskResize", "Validate storage pool expansion using resize-disk option", nil, 0)
-	})
-
-	var contexts []*scheduler.Context
-	stepLog := "has to schedule apps, and expand it by resizing a disk"
-	It(stepLog, func() {
-		log.InfoD(stepLog)
-		contexts = make([]*scheduler.Context, 0)
-
-		for i := 0; i < Inst().GlobalScaleFactor; i++ {
-			contexts = append(contexts, ScheduleApplications(fmt.Sprintf("poolexpand-%d", i))...)
-		}
+var contexts []*scheduler.Context
+var poolIDToResize string
+var poolToBeResized *api.StoragePool
+var targetSizeInBytes uint64
+var originalSizeInBytes uint64
 
+var _ = Describe("{StoragePoolExpandDiskResize}", func() {
+	BeforeEach(func() {
+		contexts = initializeContexts()
```

Comment:
StartTorpedoTest call is missing. This should be the first call in the test so that testCaseID is created and all logs are posted to that ID